tailcfg/tailcfg.go

```diff
@@ -2090,13 +2109,31 @@ type DisplayMessage struct {
 	Text string
```

Comment:
It would be great to document the expected length and style for `Text`

Reply:
Done, similarly to Title